src/renderer/components/dock/log.store.ts

```diff
@@ -45,10 +45,12 @@ export class LogStore {
     autorun(() => {
       const { selectedTab, isOpen } = dockStore;
 
-      if (isLogsTab(selectedTab) && isOpen) {
-        this.refresher.start();
-      } else {
-        this.refresher.stop();
+      if (selectedTab.kind === TabKind.TERMINAL) {
+        if (isOpen) {
+          this.refresher.start();
+        } else {
+          this.refresher.stop();
```

Comment:
If you don't want to change behaviour this needs to be called for all other tab kinds too.

Reply:
Good catch, thanks.